apptools/apptools-ios-tests/tests.full.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="Crosswalk App Tools" name="apptools-ios-tests">
+    <set name="CLI" type="pyunit">
+      <testcase component="Crosswalk App Tools/CLI" execution_type="auto" id="Crosswalk_pkgName" priority="P1" purpose="iOS - Validate if the App Tool rules about package name" status="approved" type="Functional" subcase="16">
```

Comment:
Should mark execution_type as manual, as currently testkit-lite still not support iOS.